frontend/coprs_frontend/coprs/forms.py

```diff
@@ -26,6 +26,7 @@
 from wtforms import ValidationError
 
 FALSE_VALUES = {False, "false", ""}
+REGEX_BOOTSTRAP_IMAGE = r"^[-\./\w]+(:\w+)?$"
```

Comment:
```suggestion
REGEX_BOOTSTRAP_IMAGE = r"^[-\.\/\w]+(:\w+)?$"
```
Should this backslash be escaped just in case or it doen't matter?

Reply:
that was meant to be an escape character for the following forward slash :-)